plugins/discovery-ec2/qa/amazon-ec2/build.gradle

```diff
@@ -66,19 +65,26 @@ integTest.enabled = false
  */
 ['KeyStore', 'EnvVariables', 'SystemProperties', 'ContainerCredentials', 'InstanceProfile'].forEach { action ->
   AntFixture fixture = tasks.create(name: "ec2Fixture${action}", type: AntFixture) {
-    dependsOn compileTestJava
-    env 'CLASSPATH', "${-> project.sourceSets.test.runtimeClasspath.asPath}"
+    dependsOn compileYamlRestTestJava
```

Comment:
This should really be `dependsOn project.sourceSets.yamlRestTest.runtimeClasspath`. Just `compileYamlRestTestJava` doesn't include a) processing resources or b) upstream runtime dependencies so we risk a race condition here.

Reply:
done https://github.com/elastic/elasticsearch/pull/60084/commits/01114f9eca06b6d834cf5c85bdc93bdc80faeb9f